### ISSUE TITLE
[FEATURE] Dockerize the web app

### ISSUE THREAD

**Issue by TripleteSumit** [narayan954/dummygram#727]
### Description

**Dockerizing this project will help developers contribute more easily. How? Let me explain. Before making any contributions, developers typically need to set up their development environment with all the project dependencies. By dockerizing this project, they can eliminate the overwhelming task of setting up the environment.**

### Screenshots

NA

### Additional information

NA

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I have checked the existing issues
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

**Comment by narayan954** [comment]
won't they now need to install docker in their system as well?

**Comment by TripleteSumit** [comment]
@narayan954  yes they need to install docker but it is Very easy to install and only command to run the application. After they do the changes and run that command again.

Thank you for assigning the above task to me I will try to compete as sson as possible.
